common/rdm/ResponderPersonality.cpp

```diff
@@ -40,13 +40,21 @@ using std::string;
 Personality::Personality(uint16_t footprint, const string &description)
     : m_footprint(footprint),
       m_description(description) {
+        // STATIC_ASSERT(description.length() <= 32);
+        // c++11:
+        // static_assert(
```

Comment:
Does this work in C++11? If so I guess we can leave it as a comment, but perhaps make the C++11 line a TODO(C++11) or something.

Reply:
Done.